src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$block-size: 100px;
+$block-color: #eee;
+$calendar-padding: 10px;
+$transform-time: 0.5s;
+
+$days:
+  "mon" 0,
+  "tue" calc((#{$block-size} + 3px)),
+  "wed" calc((#{$block-size} + 3px) * 2),
+  "thu" calc((#{$block-size} + 3px) * 3),
+  "fri" calc((#{$block-size} + 3px) * 4),
+  "sat" calc((#{$block-size} + 3px) * 5),
+  "sun" calc((#{$block-size} + 3px) * 6);
```

Comment:
don't repeat yourself. you can just set the numbers and make `calc `in the cycle